metadata/com.microsoft.sqlserver/mssql-jdbc/index.json

```diff
@@ -37,7 +37,16 @@
       "12.8.0.jre11",
       "12.8.0.jre8",
       "12.8.1.jre11",
-      "12.8.1.jre8"
+      "12.8.1.jre11",
+      "12.8.1.jre8",
+      "12.10.1.jre11",
+      "12.10.1.jre8",
+      "13.1.0.jre11-preview",
+      "13.1.0.jre8-preview",
+      "13.1.1.jre11-preview",
+      "13.1.1.jre8-preview",
+      "13.2.0.jre11",
+      "13.2.0.jre8"
```

Comment:
@dnestoro are all of these libraries tested when we run our gates or only the latest one?

Reply:
When we run this periodic job for scanning, we test all new versions that you marked in the comment. Once merged, we only test ["original" version](https://github.com/oracle/graalvm-reachability-metadata/blob/c0d26a8694e6d1e04126898630388aa732d95abb/metadata/com.microsoft.sqlserver/mssql-jdbc/index.json#L5) (the version that metadata was originally provided for). Since tests and metadata already work for all versions in the list, we can pick any of them, but the existing infrastructure (before we introduced this task) used original version and I kept that.